tests/console/glibc_sanity.pm

```diff
@@ -20,12 +20,13 @@ use warnings;
 use testapi;
 use Utils::Architectures;
 use utils 'zypper_call';
+use version_utils qw(is_sle is_leap);
 
 sub run {
     select_console 'root-console';
 
     my $libcstr = 'GNU C Library';
-    if (is_x86_64) {
+    if (is_x86_64 && (!is_sle('16+') || is_leap("16+"))) {
```

Comment:
`!` will not work for both is_sle and is_leap without parentheses

Reply:
Updated, new VR: https://openqa.suse.de/tests/16594198